### PR TITLE
activation functions and elipses changed

### PR DESCRIPTION
**Pull request by ghost**
This PR has the following changes:

- Activation Functions changes to LeakyReLU with alpha 0.01 
- Elipses are changes to slices for compatibility reasons with TF1 frozen graph

### CODE REVIEW

**Review by hamzarawal-kt — APPROVED** [review]

File: models/common.py [hunk 1]
```diff
@@ -123,7 +121,7 @@ def __init__(self, c1, c2, n=1, shortcut=True, g=1, e=0.5):  # ch_in, ch_out, nu
         self.cv3 = nn.Conv2d(c_, c_, 1, 1, bias=False)
         self.cv4 = Conv(2 * c_, c2, 1, 1)
         self.bn = nn.BatchNorm2d(2 * c_)  # applied to cat(cv2, cv3)
-        self.act = nn.LeakyReLU() #nn.SiLU()
+        self.act = nn.LeakyReLU()
```
Comment (faisal-kt) [inline models/common.py:124]: what's the change here?

Comment (ghost) [reply models/common.py:124]: removed the comment #nn.SiLU. Which was the previously used activation function

Comment (hamzarawal-kt) [reply models/common.py:124]: removed unwanted comment

**Review by faisal-kt — APPROVED** [review]
/approve

File: models/tf.py [hunk 2]
```diff
@@ -437,7 +437,7 @@ def predict(self,
 
         # Add TensorFlow NMS
         if tf_nms:
-            boxes = self._xywh2xyxy(x[0][..., :4])
+            boxes = self._xywh2xyxy(x[0][:, :, :4])
             probs = x[0][:, :, 4:5]
             classes = x[0][:, :, 5:]
             scores = probs * classes
```
Comment (TabasherArif) [inline models/tf.py:443]: please remove unwanted comments from this file as well

Comment (ghost) [reply models/tf.py:443]: Its already merged. And it third-party code comments.

**Pull request merged**